### PR TITLE
TCG-130: Archive 2018 content

### PR DESCRIPTION
**Pull request by santi91**
https://zoocha.atlassian.net/browse/TCG-130

 New module with hook_update_N to archive all modules from 2018. We understand archive as unpublishing because Workbench moderation is not enabled for any content types.

No manual or Features changes.

DB changes: `drush updb` to apply the update.

Test: Check all 2018 created events or sponsors are unpublished.

### CODE REVIEW

File: docroot/modules/custom/zoocha_config/zoocha_config.module [hunk 1]
```diff
@@ -0,0 +1 @@
+<?php
```
Comment (JoeKersey) [inline docroot/modules/custom/zoocha_config/zoocha_config.module]: Just to let you know, in Drupal 8 the .module file isn't required. We'll probably use it at some point though so it's worth keeping.

File: docroot/modules/custom/zoocha_config/zoocha_config.install [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * @file
+ * Update hooks for the zoocha_config module.
+ */
+
+/**
+ * Archive 2018 content.
+ */
+function zoocha_config_update_8002(&$sandbox) {
```
Comment (JoeKersey) [inline docroot/modules/custom/zoocha_config/zoocha_config.install]: What happened to 8001? 😛

File: docroot/modules/custom/zoocha_config/zoocha_config.module (hunk 1 above)
Comment (santi91) [reply docroot/modules/custom/zoocha_config/zoocha_config.module]: Oh didn't know that. Will keep it in mind.

File: docroot/modules/custom/zoocha_config/zoocha_config.install (hunk 2 above)
Comment (santi91) [reply docroot/modules/custom/zoocha_config/zoocha_config.install]: Not sure why but the hook_update_N() wasn't executed if the value was 8001!

**Pull request merged**